pythonProject1/main.py

```diff
@@ -18,19 +17,4 @@
         answer = solution
     is_primal = True
     solution += 1
```

Comment:
Variables outside of functions or classes should be consts. (All Caps)

Reply:
fuck off